### PR TITLE
add Automizely

### PR DESCRIPTION
**Pull request by gemini133**

### CODE REVIEW

File: src/technologies/a.json [hunk 1]
```diff
@@ -3155,6 +3155,22 @@
     "scriptSrc": "/wp-content/plugins/autoptimize/.+\\.js(?:\\?ao_version=(\\d+(?:\\.\\d+)+))?\\;version:\\1",
     "website": "https://autoptimize.com"
   },
+  "Automizely": {
+    "cats": [
+      32
+    ],
+    "description": "Install 1 free app to replace 30+ paid apps. Email marketing, sales pop ups, web push, cart recovery",
```
Comment (nurbek91) [inline src/technologies/a.json:3162]: Description: Automizely creates and manages enterprise-level marketing automation systems including contact and CRM mappings, lead funnels, email nurture, lead-generating pages, and blog posts, and website integrations.

File: src/technologies/a.json [hunk 2]
```diff
@@ -3155,6 +3155,22 @@
...
+    "icon": "Automizely.png",
```
Comment (nurbek91) [inline src/technologies/a.json:3163]: Icon [Automizely.svg.zip](https://github.com/AliasIO/wappalyzer/files/7601804/Automizely.svg.zip)

File: src/technologies/a.json [hunk 3]
```diff
@@ -3155,6 +3155,22 @@
...
+    "pricing": [
+      "free"
+    ],
+    "requires": [
+      "WordPress",
+      "Shopify"
+    ],
+    "scriptSrc": "widgets.automizely.com",
```
Comment (nurbek91) [inline src/technologies/a.json:3171]: "scriptSrc": "\\.automizely.\\com/",

**Review by nurbek91 — COMMENTED** [review]
Hey there, thanks!!
examples:
https://yunomi.life/
https://www.retromodding.com/
https://www.aftership.com/
https://safariland.com/
https://agacistore.com/

**Comment by gemini133** [comment]
thanks for your review. please let me know if any change required!

**Comment by nurbek91** [comment]
No problem mate!!
If you are sure that the products are intended for Shopify and Wordpress, you can probably leave.
Pricing policy needs to be looked at. Probably their products are worth something.

We have many categories. If you understand the examples and find out in which cases the website uses what, you can add them separately. But that's up to you. Thanks!!

examples list: https://publicwww.com/websites/%22automizely.com%22/2

Automizely Marketing
Automizely Reviews
Automizely Loyalty
Automizely Page Builder

File: src/technologies/a.json [hunk 4]
```diff
@@ -3155,6 +3155,22 @@
...
     "website": "https://autoptimize.com"
   },
+  "Automizely": {
+    "cats": [
+      32
+    ],
+    "description": "Automizely creates and manages enterprise-level marketing automation systems including contact and CRM mappings, lead funnels, email nurture, lead-generating pages, and blog posts, and website integrations.",
+    "icon": "Automizely.svg",
+    "pricing": [
+      "free"
```
Comment (nurbek91) [inline src/technologies/a.json:3165]: freemium instead free

**Comment by gemini133** [comment]
I think it's good to go?

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**